test/e2e_node/cpu_manager_test.go

```diff
@@ -1066,6 +1069,133 @@ func runCPUManagerTests(f *framework.Framework) {
 		waitForContainerRemoval(ctx, pod.Spec.Containers[0].Name, pod.Name, pod.Namespace)
 	})
 
+	ginkgo.It("should assign packed CPUs with distribute-cpus-across-numa disabled and pcpu-only policy options enabled", func(ctx context.Context) {
```

Comment:
on one hand this is redundant (to the letter of the def) but I prefer to have it explicit